style.css

```diff
@@ -368,131 +136,69 @@ li {
 /* second section */
 
 .work-image {
-  align-items: center;
-  padding: 4px 0;
   width: 77px;
   height: 40px;
-  left: 61.5px;
-  top: 64px;
-  margin-top: 80%;
-  position: static;
-}
-
-.main-content-area {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 3461px;
-  left: 0;
-  margin-top: -180px;
-}
-
-.title-container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 0 24px;
-  position: static;
-  height: 145px;
-  left: 0;
-  top: 120px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 40px 0;
+  margin: 0 33%  80px 33%;
 }
 
 .title-container h2 {
   height: 104px;
-  left: 24px;
-  top: 0;
   font-family: Roboto, sans-serif;
   font-style: normal;
   font-weight: 600;
   font-size: 40px;
   line-height: 52px;
-  display: flex;
   align-items: center;
   color: #9490ff;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-  margin: 24px 0;
+  margin: 0 24px;
 }
 
 .title-container img {
-  height: 0;
   width: 80%;
-  left: 0;
-  right: 0;
-  top: calc(50% - 0px / 2 + 0.5px);
+  margin: 33px 24px;
+  height: 0;
   border: 1px solid #646369;
 }
 
 .main-body {
   background-image: url(IMG1/background-image2.png);
-  height: 2617.21px;
+  background-repeat: no-repeat;
 }
```

Comment:
- [ ] Good job setting a background for this part but you are missing another background image that should be there please fix that.

![image](https://user-images.githubusercontent.com/59416136/155729759-9e3f689a-baff-424e-a370-77acde0b927c.png)